app/bundles/LeadBundle/Controller/CompanyController.php

```diff
@@ -96,17 +96,22 @@ public function indexAction(Request $request, PageHelperFactoryInterface $pageHe
         $companyIds = array_keys($companies);
         $leadCounts = (!empty($companyIds)) ? $model->getRepository()->getLeadCount($companyIds) : [];
 
+        /** @var \Mautic\UserBundle\Model\UserModel $userModel */
+        $userModel           = $this->getModel('user');
```

Comment:
Add `UserModel  user$userModel` as an explicit dependency to the `indexAction` method.